debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/VirtualThreadsCommand.java

```diff
@@ -72,43 +74,34 @@ public void run(String command, String[] args, Context context, PrintStream out)
 	 * @param out the PrintStream to write output to
 	 */
 	private static void displayVirtualThreads(J9JavaVMPointer vm, PrintStream out) throws CorruptDataException, NoSuchFieldException {
-		PointerPointer mainVirtualThread = vm.liveVirtualThreadList();
-		if (mainVirtualThread.isNull()) {
-			return;
-		}
-
-		J9ObjectPointer root = J9ObjectPointer.cast(mainVirtualThread.at(0));
-		if (root.isNull()) {
-			return;
-		}
-
-		UDATA linkNextOffset = vm.virtualThreadLinkNextOffset();
-		/*
-		 * liveVirtualThreadList is a circular doubly-linked list storing all the live virtual threads.
-		 * The root node is a dummy virtual thread marking the start and the end of the list.
-		 */
-		J9ObjectPointer node = ObjectReferencePointer.cast(root.addOffset(linkNextOffset)).at(0);
-		J9ObjectFieldOffset nameOffset = J9ObjectHelper.getFieldOffset(node, "name", "Ljava/lang/String;");
-		J9ObjectFieldOffset contOffset = J9ObjectHelper.getFieldOffset(node, "cont", "Ljdk/internal/vm/Continuation;");
-		J9ObjectPointer cont = J9ObjectHelper.getObjectField(node, contOffset);
-		J9ObjectFieldOffset vmRefOffset = J9ObjectHelper.getFieldOffset(cont, "vmRef", "J");
-
 		String addressFormat = "0x%0" + (UDATA.SIZEOF * 2) + "x";
 		String outputFormat = "!continuationstack " + addressFormat
 				+ " !j9vmcontinuation " + addressFormat
 				+ " !j9object %s (Continuation) !j9object %s (VThread) - %s%n";
-		while (!node.eq(root)) {
-			J9ObjectPointer name = J9ObjectHelper.getObjectField(node, nameOffset);
-			cont = J9ObjectHelper.getObjectField(node, contOffset);
-			long vmRef = J9ObjectHelper.getLongField(cont, vmRefOffset);
-			out.format(
-					outputFormat,
-					vmRef,
-					vmRef,
-					cont.getHexAddress(),
-					node.getHexAddress(),
-					J9ObjectHelper.stringValue(name));
-			node = ObjectReferencePointer.cast(node.addOffset(linkNextOffset)).at(0);
+		MM_GCExtensionsPointer extensions = GCBase.getExtensions();
+		UDATA linkOffset = extensions.accessBarrier()._continuationLinkOffset();
+		MM_ContinuationObjectListPointer continuationObjectLits = extensions.continuationObjectLists();
```

Comment:
Please fix this variable name; I expect you intended the name to end with "Lists", not "Lits".